app/controllers/helpers/PackageInformationHelper.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.helpers
+
+import forms.declaration.PackageInformation
+import models.requests.JourneyRequest
+
+object PackageInformationHelper {
+
+  def allCachedPackageInformation(itemId: String)(implicit request: JourneyRequest[_]): Seq[PackageInformation] =
+    maybePackageInformation(itemId).getOrElse(List.empty)
+
+  def singleCachedPackageInformation(id: String, itemId: String)(implicit request: JourneyRequest[_]): PackageInformation =
+    maybePackageInformation(itemId).flatMap(_.find(_.id == id)).getOrElse(PackageInformation(id, None, None, None))
```

Comment:
This struck me as a little odd. If no package with `id` exists, why do we return a 'fake' package rather than a None? :thinking:

Reply:
Fixed. Not sure why this was originally implemented like this, other than it seemed to make the unit testing simpler (but also basically incorrect).